pages/event-decoder.page.tsx

```diff
@@ -1,4 +1,5 @@
 import { Interface } from '@ethersproject/abi';
+import Img from 'next/image';
```

Comment:
```suggestion
import Image from 'next/image';
```

Let's keep `<img>` and `<Image>` to avoid confusion.